### PR TITLE
Collection Tab Install

### PR DESCRIPTION
**Pull request by MilanPospisil**
Tab install in Collections.

image

### CODE REVIEW

File: frontend/hub/collections/components/CodeBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+export interface CodeBlockProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the code block */
+  children?: React.ReactNode;
+  /** Additional classes passed to the code block wrapper */
+  className?: string;
+  /** Actions in the code block header. Should be wrapped with CodeBlockAction. */
+  actions?: React.ReactNode;
+}
+
+export const CodeBlock: React.FunctionComponent<CodeBlockProps> = ({
```
Comment (nixocio) [inline frontend/hub/collections/components/CodeBlock.tsx:14]: Are we using this component? Could we just leverage the native PF ones?

File: frontend/hub/collections/CollectionPage/collection-info.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.install-title {
+  min-width: 100px;
```
Comment (nixocio) [inline frontend/hub/collections/CollectionPage/collection-info.css:2]: Not used?

File: frontend/hub/collections/CollectionPage/collection-info.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+.download-button {
+  padding-left: 0;
+}
+
+.info-panel {
+  padding: 8px;
+}
+
+.hub-collection-download-alert {
```
Comment (nixocio) [inline frontend/hub/collections/CollectionPage/collection-info.css:37]: Not used?

File: frontend/hub/collections/CollectionPage/collection-info.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  width: 100%;
+}
+
+.hub-collection-download-alert > h4 {
```
Comment (nixocio) [inline frontend/hub/collections/CollectionPage/collection-info.css:41]: Not used?

**Review by nixocio — CHANGES_REQUESTED** [review]

File: frontend/hub/collections/components/CodeBlock.tsx (hunk 1 above)
Comment (MilanPospisil) [reply frontend/hub/collections/components/CodeBlock.tsx:14]: You are right. It seems my IDE even imported react version of CodeBlock, so Im not using this CodeBlock from old code. I deleted this component. I dont see any reason why not use classic react CodeBlock.

**Comment by MilanPospisil** [comment]
Note - problems with visualisation in dark mode.

**Review by nixocio — APPROVED** [review]
@MilanPospisil, is there a related issue?

**Comment by MilanPospisil** [comment]
> @MilanPospisil, is there a related issue?

After rebase, tests are ok. Mergning.

**Pull request merged**